src/open_producten/producttypen/serializers/producttype.py

```diff
@@ -352,10 +350,6 @@ def create(self, validated_data):
             ParameterSerializer,
         )
 
-        product_type.set_current_language("nl")
```

Comment:
Is this no longer needed?

Reply:
Yes, passing the values to the create call also works, it uses the default language.